samples/https/README.md

```diff
@@ -0,0 +1,23 @@
+# HTTPS service
+
+This sample consists of an NGINX service which can encrypt traffic using HTTPS.
+
+To use it:
+
+1. Generate certificates
+
+    You need to have openssl installed to run these commands:
```

Comment:
```suggestion
    You need to have [OpenSSL](https://www.openssl.org/) installed to run these commands:
```

Reply:
You need to install [OpenSSL](https://www.openssl.org/) if not already installed: